api/content-script/listeners/settings-updated.ts

```diff
@@ -0,0 +1,17 @@
+import { Settings, SETTINGS_KEY } from "@/contexts/SettingsContext.tsx";
+import { ApiUtils } from "@/api/background/utils.ts";
+
+export const watchSettingsUpdated = () => {
+  storage.watch<Settings>(SETTINGS_KEY, (newValue, oldValue) => {
+    if (newValue?.networkId !== oldValue?.networkId) {
+      window.postMessage(
+        ApiUtils.createApiResponse("kas_networkChanged", [newValue?.networkId]),
+        window.location.origin,
+      );
+      window.postMessage(
+        ApiUtils.createApiResponse("kas_accountChanged", null),
```

Comment:
It would be better to return address if the account is connected to website in the target network.

Reply:
3ee36a59e314208f24a46de7206e567cae2a8ca1

Reply:
e14bd84c19153a7d0d2a500fffbdd0c9303e56ab